packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -196,8 +196,11 @@ const TESTS: [string, any, Record<string, any>?][] = [
       { test: 'include', expected: decodeUrl(configHref('supportUrl', 'cad-reminder-first', 'support')) },
       { test: 'include', expected: 'alt="Firefox logo"' },
       { test: 'include', expected: 'alt="Devices"' },
-      { test: 'include', expected: `alt="Download Firefox on the App Store"` },
-      { test: 'include', expected: `alt="Download Firefox on Google Play"` },
+      // FIXME: these tests fail as all email tests utilize the same testing parameters
```

Comment:
I think you can fix this by using `updateTemplateValues`

Reply:
Updated, thanks!